### PR TITLE
🎉 New Source: LinkedIn Pages

### PR DESCRIPTION
**Pull request by jscottpolevault**
## What
- Added a LinkedIn Company Pages connector

## How
- The Python CDK

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [X] Secrets in the connector's spec are annotated with `airbyte_secret`
- [X] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [X] Code reviews completed
- [X] Documentation updated
    - [X] Connector's `README.md`
    - [X] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [X] `docs/SUMMARY.md`
    - [X] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [X] `docs/integrations/README.md`
    - [X] `airbyte-integrations/builds.md`
- [X] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

_No Custom Unit Tests Added_

</details>

<details><summary><strong>Integration</strong></summary>

PASSED

</details>

<details><summary><strong>Acceptance</strong></summary>

21 PASSED (ALL)

</details>

### CODE REVIEW

**Comment by marcosmarxm** [comment]
@jscottpolevault can you post the output image of integration tests?

**Comment by jscottpolevault** [comment]
Is this what you're asking for @marcosmarxm?
Screen Shot 2022-05-27 at 7 52 53 AM

File: .vscode/settings.json [hunk 1]
```diff
@@ -31,6 +31,9 @@
...
   "[json]": {
     "editor.formatOnSave": true,
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
-  }
+    "editor.defaultFormatter": "vscode.json-language-features"
+  },
+  "python.analysis.extraPaths": [
+    "./airbyte-cdk/python"
+  ]
 }
```
Comment (marcosmarxm) [inline .vscode/settings.json:39]: can you remove this change?

File: CODE_OF_CONDUCT 2.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Code of conduct
+View in [docs.airbyte.io](https://docs.airbyte.io/contributing-to-airbyte/code-of-conduct)
```
Comment (marcosmarxm) [inline CODE_OF_CONDUCT 2.md:2]: aslo this document

File: CONTRIBUTING 2.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Contributing
```
Comment (marcosmarxm) [inline CONTRIBUTING 2.md:1]: and remove this too.

File: airbyte-integrations/connectors/source-linkedin-pages/acceptance-test-config.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
+  full_refresh:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-linkedin-pages/acceptance-test-config.yml:19]: EOF

File: airbyte-integrations/connectors/source-linkedin-pages/integration_tests/abnormal_state.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  "shares": {
+    "lastModified": "2050-01-01"
+  },
+  "total_follower_count": {
+    "end_date": "2050-01-01"
+  },
+  "ugc_posts": {
+    "end_date": "2050-01-01"
+  }
+}
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-linkedin-pages/integration_tests/abnormal_state.json:23]: EOF

File: airbyte-integrations/connectors/source-linkedin-pages/integration_tests/sample_config.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
+  "org_id": 12345678,
+  "credentials": {
+    "auth_method": "access_token",
+    "access_token": "drAL4JXBJ6v1qaU9iWargosra6ibiw0ZWEdMnC0ZizeD1gLRQP6u1pkQ"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-linkedin-pages/integration_tests/sample_config.json:5]: This is not a valid token right?

File: codecov 2.yml [hunk 7]
```diff
@@ -0,0 +1,18 @@
+codecov:
+  notify:
+    require_ci_to_pass: no
+
+coverage:
+  status:
+    patch:
```
Comment (marcosmarxm) [inline codecov 2.yml:7]: remove this.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
thanks for the contribution, sorry the long delay. Requested some changes.

File: airbyte-integrations/connectors/source-linkedin-pages/integration_tests/sample_config.json (hunk 6 above)
Comment (jscottpolevault) [reply airbyte-integrations/connectors/source-linkedin-pages/integration_tests/sample_config.json:5]: no it is not, but i'll make it more obvious =)

**Comment by jscottpolevault** [comment]
> [CLA assistant check](https://cla-assistant.io/airbytehq/airbyte?pullRequest=13098) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/airbytehq/airbyte?pullRequest=13098) before we can accept your contribution.**1** out of **2** committers have signed the CLA.✅ jscottpolevault❌ Jordan Scott
> 
> **Jordan Scott** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/airbytehq/airbyte?pullRequest=13098) it.

@marcosmarxm - Sorry once again, but I had 2 github accounts and realized I didn't need both, so I deleted one of the accounts. I mistakenly "contributed" to this from both accounts, but now the CLA says it's not signed by all contributors... but it's just me.

Let me know if I can do anything on my end to fix this.

**Comment by marcosmarxm** [comment]
@jscottpolevault can you paste the integration/acceptance and unit test screenshoots too?

**Comment by jscottpolevault** [comment]
@marcosmarxm - I don't know anything about the unit tests and I thought the integration/acceptance test was the screenshot I added above, right?

I just tried running `python -m pytest -s unit_tests` and all 12 failed. I'm sure I need to change the test_source.py file, but I can't find documentation to help me through it.

Screenshot below:
Screen Shot 2022-06-01 at 4 48 20 PM

**Comment by marcosmarxm** [comment]
Sorry the long delay @jscottpolevault I'm review/publish max tomorrow this contribution

**Comment by marcosmarxm** [comment]
The command to publish without integration account was removed. I'll work this week to implement it back and publish your contribution.

**Comment by jscottpolevault** [comment]
The connector needs pagination, which I can add in before or after it's published. I don't want to mess with things now while you're finishing things up.

**Comment by sherifnada** [comment]
Could you fix the diff?

**Comment by marcosmarxm** [comment]
> Could you fix the diff?

My bad! It's fixed now.

**Comment by marcosmarxm** [comment]
Sorry the long delay to merge this @jscottpolevault I made some changes and removed 3 streams (posts and shares because they are legacy, and page_statistics) feel free to add them again in the future.

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**